### PR TITLE
Refactor resource image_export and documentation

### PR DESCRIPTION
**Pull request by Diptipowervs**
### Community Note

* Please vote on this pull request by adding a 👍 [reaction](https://blog.github.com/2016-03-10-add-reactions-to-pull-requests-issues-and-comments/) to the original pull request comment to help the community and maintainers prioritize this request
* Please do not leave "+1" or other comments that do not add relevant new information or questions, they generate extra noise for pull request followers and do not help prioritize the request

Relates OR Closes #0000

Output from acceptance testing:

```
$ make testacc TESTARGS='-run=TestAccXXX'

...
```

### CODE REVIEW

File: ibm/service/power/resource_ibm_pi_image_export.go [hunk 1]
```diff
@@ -31,46 +29,45 @@ func ResourceIBMPIImageExport() *schema.Resource {
...
 				Type:        schema.TypeString,
+				Required:    true,
+			},
+			Arg_ImageBucketRegion: {
 				Description: "Cloud Object Storage region",
 				ForceNew:    true,
 				Required:    true,
+				Type:        schema.TypeString,
+			},
+			Attr_ImageID: {
```
Comment (ismirlia) [inline ibm/service/power/resource_ibm_pi_image_export.go:58]: This is the wrong constant. Before and after the refactor the value of the constants should be the same. The only thing that should change is the name. The constant here should be called Arg_ImageID with value "pi_image_id"

Comment (Diptipowervs) [reply ibm/service/power/resource_ibm_pi_image_export.go]: Added

**Review by michaelkad — APPROVED** [review]
lgtm

**Comment by michaelkad** [comment]
Opened #129

**Pull request closed without merging**